tools/dhis2-package-exporter/package_exporter.py

```diff
@@ -925,6 +925,8 @@ def main():
     my_parser.set_defaults(verbose=False)
     my_parser.add_argument('-od', '--only_dashboards', dest='only_dashboards', action='store_true')
     my_parser.set_defaults(only_dashboards=False)
+    my_parser.add_argument('-efo', '--export_for_oslo', dest='export_for_oslo', action='store_true')
```

Comment:
I think we should use a more descriptive name for the argument flag. Since this is removing org units from the export, I suppose it should be something like `export_without_ous`.

Reply:
Let's go with export full objects as Phil suggested